tasks/install.yml

```diff
@@ -1,38 +1,24 @@
 ---
-- name: 'Download binary from https://releases.mattermost.com'
-  get_url: 
-    url: 'https://releases.mattermost.com/{{ mattermost_version }}/mattermost-team-{{ mattermost_version }}-linux-amd64.tar.gz'
-    dest: "{{ global_cache_dir | mandatory }}/"
-  when: not mattermost_enterprise
-  become: false
-  delegate_to: localhost
 
-- name: 'Download enterprise version binary from https://releases.mattermost.com'
-  get_url: 
-    url: 'https://releases.mattermost.com/{{ mattermost_version }}/mattermost-{{ mattermost_version }}-linux-amd64.tar.gz'
-    dest: "{{ global_cache_dir | mandatory }}/"
-  when: mattermost_enterprise
-  become: false
-  delegate_to: localhost
+- name: Check if the desired mattermost version exists
+  stat:
+    path: /opt/mattermost-versions/{{ mattermost_version }}
```

Comment:
Won't this task fail if `/opt/mattermost-versions` doesn't exist, i.e. the first time this role is run on a server?

Reply:
Shouldn't stat can handle folders that don't exist and can therefore be used to check whether a folder exists at all.

I will test it later, though.

Reply:
I just tested it and it works like desired.

Reply:
Okay, great. I just tested it using the `stat` command, which returned 1 in such a case.